src/Forms/TextCheckboxGroupField.php

```diff
@@ -46,19 +46,24 @@ public function performReadonlyTransformation()
 
             $field->setTemplate(CompositeField::class);
             $field->setTitle('Title');
-
-            $field->replaceField('Title', LiteralField::create(
-                'Title',
-                $field->fieldByName('Title')->Value()
-            ));
+            
+            $titleField = $field->fieldByName('Title');
+            if ($titleField) {
```

Comment:
seems like it would make more sense to change this PR so that we test for the existence of the field within the replaceField function instead?

Reply:
`replaceField` actually fails gracefully. It's the unguarded of `$titleField->Value()` that blows up.

Reply:
ah ok, so what's happening is it's going null->Value() ?

Reply:
Yup. 💥